### ISSUE TITLE
Select multiple items

### ISSUE THREAD

**Issue by UndermanReal** [TurboWarp/scratch-gui#1067]
![image](https://github.com/TurboWarp/scratch-gui/assets/131917462/484f9e33-2f23-44b1-8904-652ad31c042e)
Like this, you should be able to select multiple blocks at the same time. 
![image](https://github.com/TurboWarp/scratch-gui/assets/131917462/134570a7-728e-48b3-aada-e6b59e0ea566)
This could be useful if your blocks are all around like this and you don't want to delete them by hand
I'd suggest using middle click to do this

**Comment by Djidjey** [comment]
Something i can suggest you actually is packing up (?) all your block, then delete them.
but i agree that something like that would be very useful
![image](https://github.com/TurboWarp/scratch-gui/assets/93743322/5e8ae7c0-0087-47d0-a123-b6eeafb227db)

**Comment by UndermanReal** [comment]
Yeah, but it would still be quicker to select them all at once

**Comment by Djidjey** [comment]
yes it's just a suggestion for you the time this function is added
